src/config.js

```diff
@@ -146,26 +188,31 @@ export default async function load(profile, options = {}) {
     path.join(__dirname, '..', 'src', 'config', `${profile}.yml`)
   );
 
-  let baseConfig = _.merge({}, defaultConfig, profileConfig);
+  let baseConfig = _.merge(
+      {},
+      defaultConfig,
+      profileConfig,
+      options.overrides || {});
 
   // extend the base config with additional parameters from files...
   let extraYamlConfigFiles = (baseConfig.config.files || []);
-  debug('Loading additional configs', extraYamlConfigFiles);
-  for (let yamlConfigFile of extraYamlConfigFiles) {
-    // Convert the path to be relative to the root of the project...
-    let yamlConfigPath = path.join(__dirname, '..', yamlConfigFile);
-
-    // Skip any config files which do not exist they are not required...
-    if (!await fs.exists(yamlConfigPath)) {
-      debug('skip config', yamlConfigPath);
-      continue;
-    }
+  if (extraYamlConfigFiles.length) {
```

Comment:
Why the check for length first? I'm sure it's needed, was just educating myself.